internal/common/model/db/Run.go

```diff
@@ -19,6 +19,7 @@ const (
 type Run struct {
 	ID      uuid.UUID `gorm:"type:uuid"`
 	Account string
+	OrgId   string
```

Comment:
For reasons that are not entirely clear to me there is a convention in the golang community to do uppercase abbreviations i.e. OrgID instead of OrgId (https://github.com/golang/lint/issues/124). I cannot say I love that but let's do that for the sake of not breaking conventions, linters, etc.

Reply:
Gotcha!  Just pushed up that change!